intel-lustre-client-server/azuredeploy.json

```diff
@@ -0,0 +1,1228 @@
+				"Standard_DS11",
+				"Standard_DS12",
+				"Standard_DS13",
+				"Standard_DS14",
+				"Standard_DS2_v2",
+				"Standard_DS3_v2",
+				"Standard_DS4_v2",
+				"Standard_DS5_v2",
+				"Standard_DS11_v2",
+				"Standard_DS12_v2",
+				"Standard_DS13_v2",
+				"Standard_DS14_v2",
+				"Standard_D11",
+				"Standard_D12",
+				"Standard_D13",
+				"Standard_D14",
+				"Standard_G1",
+				"Standard_G2",
+				"Standard_G3",
+				"Standard_G4",
+				"Standard_G5",
+				"Standard_GS1",
+				"Standard_GS2",
+				"Standard_GS3",
+				"Standard_GS4",
+				"Standard_GS5",
+				"Standard_A2",
+				"Standard_A3",
+				"Standard_A4",
+				"Standard_A5",
+				"Standard_A6",
+				"Standard_A7",
+				"Standard_A8",
+				"Standard_A9",
+				"Standard_A10",
+				"Standard_A11",
+				"Standard_A1",
+				"Standard_D1",
+				"Standard_D1_v2",
+				"Standard_DS1",
+				"Standard_DS1_v2"
+			]
+		},
+		"clientCount": {
+			"type": "int",
+			"defaultValue": 2,
+			"allowedValues": [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50],
+			"metadata": {
+				"description": "Number of Lustre clients"
+			}
+		},
+		"clientFilesystemName": {
+			"type": "string",
+			"defaultValue": "scratch",
+			"metadata": {
+				"description": "Name of the Lustre filesystem exposed by the Lustre MGS node"
+			}
+		},
+		"mgsIpAddress": {
+			"type": "string",
+			"defaultValue": "10.1.0.4",
+			"metadata": {
+				"description": "IP address of the Lustre MGS node"
+			}
+		},
+		"clientStorageAccountType": {
+			"type": "string",
+			"defaultValue": "Standard_LRS",
+			"allowedValues": [
+				"Standard_LRS",
+				"Premium_LRS"
+			],
+			"metadata": {
+				"description": "Storage account type (e.g. Premium_LRS or Standard_LRS). Make sure to select Premium_LRS only when using DS-series or GS-series Virtual Machines."
+			}
+		},
+		"existingVnetResourceGroupNameForClient": {
+			"type": "string",
+			"metadata": {
+				"description": "Existing Virtual Network Resource Group where Lustre servers are deployed"
+			}
+		},
+		"existingVnetNameForClient": {
+			"type": "string",
+			"defaultValue": "vnet-lustre",
+			"metadata": {
+				"description": "Existing Virtual Network name (e.g. vnet-lustre)"
+			}
+		},
+		"existingSubnetClientsName": {
+			"type": "string",
+			"defaultValue": "subnet-lustre-clients",
+			"metadata": {
+				"description": "Lustre clients will be deployed into this subnet within the existing Virtual Network"
+			}
+		}
+	},
+	"variables": {
+		"imageReferenceLustre": {
+			"publisher": "intel",
+			"offer": "lustre-cloud-edition-cv-image",
+			"sku": "eval-lustre-2-7",
+			"version": "latest"
+		},
+		"ossAvailabilitySetSettings": {
+			"name": "[concat(parameters('vmNamePrefix'),'oss')]",
+			"faultDomainCount": "3",
+			"updateDomainCount": "5"
+		},
+		"baseUrl": "[concat(parameters('artifactsBaseUrl'),'/')]",
```

Comment:
Look into organizing files according to the guidelines https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#files-folders-and-naming-conventions